vegafusion-common/src/data/table.rs

```diff
@@ -271,6 +272,13 @@ impl VegaFusionTable {
         }
     }
 
+    #[cfg(feature = "pyarrow")]
+    pub fn from_arrow_c_stream(table: &Bound<PyAny>) -> PyResult<Self> {
+        let pytable = pyo3_arrow::PyTable::from_py_object_bound(table.as_borrowed())?;
```

Comment:
If you keep this method the same to only support `PyTable`, `PyTable` implements `FromPyObject`, so you can make it the parameter directly.
```suggestion
    pub fn from_arrow_c_stream(table: pyo3_arrow::PyTable) -> PyResult<Self> {
```